mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -73,28 +83,58 @@ def _state_name(self):
         readonly=True,
         default="NEW"
     )
-    date = fields.Date(
-        'Date',
+
+    state_id = fields.Many2one(
+        'mgmtsystem.nonconformity.state',
+        'State',
+        default=_default_state)
+
+    # Compute data
+    number_of_nonconformities = fields.Integer(
+        '# of nonconformities', readonly=True, default=1)
+    age = fields.Integer('Age', readonly=True,
+                         compute='_compute_age', store=True)
```

Comment:
This can't have `store=True`, since the computation depends on current date.
As is will store zero all the time.